### PR TITLE
[sui-studio] Compatibilty with windows

### PR DESCRIPTION
**Pull request by davidbarna**

### CODE REVIEW

File: packages/sui-bundler/webpack.config.dev.js [hunk 1]
```diff
@@ -5,6 +5,17 @@ const LoaderUniversalOptionsPlugin = require('./plugins/loader-options')
...
 require('./shared/shims')
 
+let pwd = process.cwd()
+
+// hack for Windows, as process.env.PWD is undefined in that environment
+// https://github.com/mrblueblue/gettext-loader/issues/18
+// Moreover, to have same caseing we need to map it.
+if (process.platform === 'win32') {
+  const [driveLetter, path] = pwd.split(':')
+  pwd = [driveLetter.toLowerCase(), path].join(':')
```
Comment (jcger) [inline packages/sui-bundler/webpack.config.dev.js]: is this really needed? I mean, windows isn't case sensitive, right?

just asking

Comment (davidbarna) [reply packages/sui-bundler/webpack.config.dev.js]: webpack is. You get millions of warnings...

File: packages/sui-bundler/webpack.config.dev.js [hunk 2]
```diff
@@ -5,6 +5,17 @@ const LoaderUniversalOptionsPlugin = require('./plugins/loader-options')
 const babelRules = require('./shared/module-rules-babel')
 require('./shared/shims')
 
+let pwd = process.cwd()
+
+// hack for Windows, as process.env.PWD is undefined in that environment
+// https://github.com/mrblueblue/gettext-loader/issues/18
+// Moreover, to have same caseing we need to map it.
+if (process.platform === 'win32') {
```
Comment (miduga) [inline packages/sui-bundler/webpack.config.dev.js]: What about extracting this in order to avoid repeating the code?

Comment (davidbarna) [reply packages/sui-bundler/webpack.config.dev.js]: On it !

Comment (davidbarna) [reply packages/sui-bundler/webpack.config.dev.js]: Done !

**Pull request merged**